x-pack/test/functional/apps/remote_clusters/ccs/remote_clusters_index_management_flow.ts

```diff
@@ -43,12 +43,14 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           return await testSubjects.isDisplayed('remoteClusterListTable');
         });
         const remotes = await pageObjects.remoteClusters.getRemoteClustersList();
+        const filteredRemotes = remotes.filter(async (remote) => {
+          return remote.remoteName === 'ftr-remote';
+        });
```

Comment:
This doesn't seem to be working as expected.  I added this line right after it;
```
console.log(JSON.stringify(filteredRemotes));
```
And the output still shows both "ftr-remote" and "local";

[{"remoteName":"ftr-remote","remoteStatus":"Connected","remoteMode":"default","remoteAddress":"localhost:9301","remoteConnectionCount":"1"},{"remoteName":"local","remoteStatus":"Connected","remoteMode":"default","remoteAddress":"localhost:9300","remoteConnectionCount":"1"}]

And so the test still fails with `Error: expected 2 to sort of equal 1`